apis/network/v1alpha3/types.go

```diff
@@ -271,14 +270,15 @@ type PublicIPAddressFormat struct {
 
 	// Tags - Resource tags.
 	// +optional
-	Tags map[string]*string `json:"tags,omitempty"`
+	Tags map[string]string `json:"tags,omitempty"`
 }
 
 // SKU of PublicIPAddress
 type SKU struct {
 	// Name - Name of sku. Possible values include: ['Standard', 'Basic']
+	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Enum=Standard;Basic
-	Name string `json:"name,omitempty"`
+	Name string `json:"name"`
```

Comment:
`spec.forProvider.sku` is optional. However, if a `spec.forProvider.sku` is specified, I think we need to have its name also specified. Thus removing the `omitempty` tag.